### DISCUSSION TITLE
製作遊戲的好幫手_pygame

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#417] [General]
之前看到有其他人分享過tkinter這個python基本的套件，就上網搜尋了一下。了解到python還有其他遊戲製作相關的package。本次要分享的pygame就是其中的一種。簡單分享一下心得，tkinter主要在GUI的繪製上可以有比較好用的方法，而pygame則適合使用在遊戲的製作上面(有比較多的圖形聲音繫結之類的需求)。另外在撰寫的邏輯上面兩者也有些許的不同。簡單來說若需要精美的GUI設計可能就不太適合pygame，而如果需要製作出生動的小遊戲則是pygame可以比較方便。
[https://www.itread01.com/p/469221.html](https://www.itread01.com/p/469221.html)
[https://ithelp.ithome.com.tw/articles/10220072](https://ithelp.ithome.com.tw/articles/10220072)

**Comment by chia-kai-chang** [comment]
IDLE就是用tkinter做出來的視窗介面喔。
![image](https://user-images.githubusercontent.com/1768891/157259734-986a64c5-1e4a-4bb6-8cd3-4d62e19be4b3.png)

**Comment by sean1325418** [comment]
以前曾經有用pygame寫過小遊戲，當時只是寫個讓球滾動的畫面，是真的很簡單容易上手
